CONTRIBUTING.rst

```diff
@@ -1,47 +1,179 @@
 .. _contributing:
 
+How to help?
+=============
+
+* If you have a usage question : create a question on `neurostars <https://neurostars.org/>`_ with tag nilearn
+
+* You think you discovered a bug ? Create `an issue <https://github.com/nilearn/nilearn/issues>`_
+  including a minimal runnable example to showcase it (preferably using Nilearn data).
```

Comment:
Maybe we could add that the issue should specify the OS and the version of Nilearn used as this is minimum information to investigate a potential bug.